### PR TITLE
default handler returns env vars and worker count

### PR DESCRIPTION
**Pull request by gfr10598**
This change is [Reviewable](https://reviewable.io/reviews/m-lab/etl/381)

### CODE REVIEW

File: cmd/etl_worker/etl_worker.go [hunk 1]
```diff
@@ -49,7 +49,13 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		http.NotFound(w, r)
 		return
 	}
-	fmt.Fprint(w, "Hello world!")
+	fmt.Fprintf(w, "NOTE: This is just one of potentially many instances.\n")
+	fmt.Fprintf(w, "Release: %s   Commit: %s\n", os.Getenv("RELEASE_TAG"), os.Getenv("COMMIT_HASH"))
+	fmt.Fprintf(w, "Workers: %d / %d\n", atomic.LoadInt32(&inFlight), maxInFlight)
+	env := os.Environ()
+	for i := range env {
+		fmt.Fprintf(w, "    %s\n", env[i])
```
Comment (stephen-soltesz) [inline cmd/etl_worker/etl_worker.go]: There's nothing sensitive in app engine env?

**Comment by stephen-soltesz** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[cmd/etl_worker/etl_worker.go, line 53 at r1](https://reviewable.io:443/reviews/m-lab/etl/381#-L2N3T1SrWDK4xOX35QK:-L2N3T1SrWDK4xOX35QL:bfvhvnw) ([raw file](https://github.com/m-lab/etl/blob/f70308d9062c0f148e9453ceced6ae159d40c765/cmd/etl_worker/etl_worker.go#L53)):*
> ```Go
> 	}
> 	fmt.Fprintf(w, "NOTE: This is just one of potentially many instances.\n")
> 	fmt.Fprintf(w, "Release: %s   Commit: %s\n", os.Getenv("RELEASE_TAG"), os.Getenv("COMMIT_HASH"))
> ```

Possibility:

This could be formatted as a url: e.g.

https://github.com/m-lab/etl/commit/feca9ece1866ae78e0bebdcdd81a653935c628a4 

or 

https://github.com/m-lab/etl/tree/feca9ece1866ae78e0bebdcdd81a653935c628a4

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/381#-:-L2N3GJU-p7CeWoOwKAo:bnfp4nl)*

**Comment by stephen-soltesz** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[cmd/etl_worker/etl_worker.go, line 53 at r1](https://reviewable.io:443/reviews/m-lab/etl/381#-L2N3T1SrWDK4xOX35QK:-L2N3rfpNSS9eNTieRUZ:bezxcx8) ([raw file](https://github.com/m-lab/etl/blob/f70308d9062c0f148e9453ceced6ae159d40c765/cmd/etl_worker/etl_worker.go#L53)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Possibility:

This could be formatted as a url: e.g.

https://github.com/m-lab/etl/commit/feca9ece1866ae78e0bebdcdd81a653935c628a4 

or 

https://github.com/m-lab/etl/tree/feca9ece1866ae78e0bebdcdd81a653935c628a4
</blockquote></details>

Without auto-formatting:

`https://github.com/m-lab/etl/commit/feca9ece1866ae78e0bebdcdd81a653935c628a4`

or

`https://github.com/m-lab/etl/tree/feca9ece1866ae78e0bebdcdd81a653935c628a4`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/381)*

**Comment by gfr10598** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

*[cmd/etl_worker/etl_worker.go, line 53 at r1](https://reviewable.io:443/reviews/m-lab/etl/381#-L2N3T1SrWDK4xOX35QK:-L2QcK_UolrrCbtHsKct:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/f70308d9062c0f148e9453ceced6ae159d40c765/cmd/etl_worker/etl_worker.go#L53)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Without auto-formatting:

`https://github.com/m-lab/etl/commit/feca9ece1866ae78e0bebdcdd81a653935c628a4`

or

`https://github.com/m-lab/etl/tree/feca9ece1866ae78e0bebdcdd81a653935c628a4`
</blockquote></details>

Done.

---

*[cmd/etl_worker/etl_worker.go, line 57 at r1](https://reviewable.io:443/reviews/m-lab/etl/381#-L2N2TLFKxT-0jwnFKga-r1-57:-L2QSmLUP-ivKRqypicy:b7jp98) ([raw file](https://github.com/m-lab/etl/blob/f70308d9062c0f148e9453ceced6ae159d40c765/cmd/etl_worker/etl_worker.go#L57)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

There's nothing sensitive in app engine env?
</blockquote></details>

There might be.  I've added a TODO and issue to add whitelist/blacklist.  I don't think we have anything sensitive yet, but should keep track of this.

---

*[cmd/etl_worker/etl_worker.go, line 286 at r2](https://reviewable.io:443/reviews/m-lab/etl/381#-L2QcEx3GG65Vp1-yGjx:-L2QcEx3GG65Vp1-yGjy:brj44qa) ([raw file](https://github.com/m-lab/etl/blob/181408dcd0c9b28bdaea5e1392ed8c3a9385e5e9/cmd/etl_worker/etl_worker.go#L286)):*
> ```Go
> 
> 	http.HandleFunc("/", handler)
> 	http.HandleFunc("/status", handler)
> ```

For some reason this isn't working.  Any idea?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/381)*

**Comment by stephen-soltesz** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

*[cmd/etl_worker/etl_worker.go, line 54 at r2](https://reviewable.io:443/reviews/m-lab/etl/381#-L2Qd1UguLe-K3iSM-31:-L2Qd1UhSCBdAYgWVwve:b-3btiyp) ([raw file](https://github.com/m-lab/etl/blob/181408dcd0c9b28bdaea5e1392ed8c3a9385e5e9/cmd/etl_worker/etl_worker.go#L54)):*
> ```Go
> 		return
> 	}
> 	fmt.Fprintf(w, "<html><body>\n")
> ```

For the future, this output could be formatted via an http template instead. May make the logic more concise.

---

*[cmd/etl_worker/etl_worker.go, line 58 at r2](https://reviewable.io:443/reviews/m-lab/etl/381#-L2Qf4-yjAW_0nFnHgoZ:-L2Qf4-yjAW_0nFnHgo_:b-y6c0ys) ([raw file](https://github.com/m-lab/etl/blob/181408dcd0c9b28bdaea5e1392ed8c3a9385e5e9/cmd/etl_worker/etl_worker.go#L58)):*
> ```Go
> 	commit := os.Getenv("COMMIT_HASH")
> 	if len(commit) >= 8 {
> 		fmt.Fprintf(w, "Release: %s <br>  Commit: <a href=\"%s\">%s</a><br>\n",
> ```

Does this link work? I expect to see `https://github.com` as part of the href, or is that baked into the `COMMIT_HASH`?

---

*[cmd/etl_worker/etl_worker.go, line 286 at r2](https://reviewable.io:443/reviews/m-lab/etl/381#-L2QcEx3GG65Vp1-yGjx:-L2Qf-RvKjAMrYcQ7p0T:bm2ounn) ([raw file](https://github.com/m-lab/etl/blob/181408dcd0c9b28bdaea5e1392ed8c3a9385e5e9/cmd/etl_worker/etl_worker.go#L286)):*
<details><summary><i>Previously, gfr10598 (Gregory Russell) wrote…</i></summary><blockquote>

For some reason this isn't working.  Any idea?
</blockquote></details>

Found it!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/381)*

**Comment by gfr10598** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[cmd/etl_worker/etl_worker.go, line 58 at r2](https://reviewable.io:443/reviews/m-lab/etl/381#-L2Qf4-yjAW_0nFnHgoZ:-L2QmVh5IdxTd9Bi9TPM:bemjmoe) ([raw file](https://github.com/m-lab/etl/blob/181408dcd0c9b28bdaea5e1392ed8c3a9385e5e9/cmd/etl_worker/etl_worker.go#L58)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Does this link work? I expect to see `https://github.com` as part of the href, or is that baked into the `COMMIT_HASH`?
</blockquote></details>

Oops.  I had it working and then broke it.  Fixed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/381)*

**Pull request merged**